src/gl/pointFeature.js

```diff
@@ -26,6 +26,7 @@ geo.gl.pointFeature = function (arg) {
       m_pixelWidthUniform = null,
       m_aspectUniform = null,
       m_dynamicDraw = arg.dynamicDraw === undefined ? false : arg.dynamicDraw,
+      m_primitiveShape = arg.primitiveShape || "triangle",
```

Comment:
Can we document above the object definition that arg can have keys such as triangle and square (and may be check if none of the above two but something else, then it should be an error)